web/client/src/components/form/TokenDisplay.jsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { Typography, Card, CardContent, Button } from "@material-ui/core";
+import { CopyToClipboard } from "react-copy-to-clipboard";
+import PropTypes from "prop-types";
+/**
+ *
+ * @param {*} param0 token being passed for display
+ * @return {div} returns a card so the token stands
+ */
+export default function TokenDisplay({ token }) {
```

Comment:
Unwrapping props directly here isn't recommended. We should do `const { token } = props` and pass `props` as the parameter.